CKEditorOptions.ascx.cs

```diff
@@ -971,9 +971,6 @@ private void BindUserGroupsGridView()
 
             lblRole.Text = Localization.GetString("lblRole.Text", ResXFile, LangCode);
             lblSelToolb.Text = Localization.GetString("lblSelToolb.Text", ResXFile, LangCode);
-            
-            lblRole = (Label)UploadFileLimits.HeaderRow.FindControl("lblRole");
-            lblSelToolb = (Label)UploadFileLimits.HeaderRow.FindControl("SizeLimitLabel");
```

Comment:
if these are removed, the next two lines need to be removed as well, right?  otherwise the toolbars get the wrong label.